app/helpers/Evaluation/EvaluationResults/SandboxResults/ISandboxResults.php

```diff
@@ -38,6 +38,12 @@ public function getUsedMemory(): int;
      */
     public function getExitCode(): int;
 
+    /**
+     * Get exit signal that examined program
```

Comment:
This seems like a bit unfinished sentence

Reply:
indee